bigtable/test_bigtable.py

```diff
@@ -65,25 +90,21 @@ def verify_instance(self, name):
     """
 
     @parameterized.expand(
-        [
-            ("SINGLE", "1.2", ["m"]),
-            ("STANDARD", "1.2", ["m"]),
-            ("HA", "1.2", ["m-0"]),
-            ("SINGLE", "1.3", ["m"]),
-            ("STANDARD", "1.3", ["m"]),
-            ("HA", "1.3", ["m-0"]),
-        ],
-        testcase_func_name=DataprocTestCase.generate_verbose_test_name)
-    def test_bigtable(self, configuration, dataproc_version, machine_suffixes):
+        buildParameters(),
```

Comment:
After removing `dataproc_version` from the parameters we may not need to override them, because they are fairly test specific and more or less static.

Let's remove `params` flag from the test and just use default parameters directly here and in all other tests.

Reply:
Done. I also updated the design doc: https://docs.google.com/document/d/1-NXCdXnJ7RnORiZu2nN05xf3kUomFQruJKhqPNuL2n8/edit#

Reply:
Thanks!